src/components/Contacts/ContactListTable.jsx

```diff
@@ -50,31 +54,68 @@ const ContactListTable = ({ contacts, deleteContact }) => {
   const contactsCopy = [...contacts];
   const sortedContacts = contactsCopy.sort(comparePerson);
 
+  const columnTitlesArray = [
+    {
+      field: 'Contact',
+      width: 160,
+      headerAlign: 'center',
+      align: 'center',
+      sortable: false
```

Comment:
should the contact name not be sortable?

Reply:
I assume this is the arrow that shows if you hover and makes sorted by ascending or descending values...

Reply:
If so, personally I would think this to be one of the most used features of the grid honestly.

Reply:
I was thinking about sorting with what we have now. Was thinking we could perhaps split up `person` back into just `givenName` and `familyName`. If we do that, think having the sorting then would be better. See clip:

https://github.com/codeforpdx/PASS/assets/14917816/c749ec9e-218e-4895-9d95-eea30d59d8a3

It'll open up more fields for filtering as well.

https://github.com/codeforpdx/PASS/assets/14917816/e1df46fd-f968-4d14-942f-a180a19925e6

The changes done in those samples are relatively simple.

Reply:
Got Person split up into First Name and Last Name and allow sorting of those fields. The clip for the update is now shown in the PR description. Let me know if you think this is good.